### ISSUE TITLE
Categories

### ISSUE THREAD

**Issue by itrests** [graphprotocol/mission-control-curator#33]
Consider adding such categories as "NFT", "Metaverse", "Gambling", "Crowdfunding", "Society", "DNS", "Mesh Networking", "Nodes" and "Mining"

**Comment by yanivtal** [comment]
Is Metaverse different from Virtual Reality?
For DNS we have Science & Tech > Software > Naming
For Mesh Networking we have Science & Tech > Telecom
For Nodes and Mining we have DeFi > Staking, but maybe that should move somewhere else?

What did you have in mind for Society?

**Comment by itrests** [comment]
yes. metaverse not same vr. for example decentraland, cryptovoxels or sanbox are a metaverses without vr suuport.
nodes and mining may be called "infrastructure" to separate it from online staking.
society is an umbrella category involving law, politics, history, psychology and other humanitarian directions.
